.github/pull_request_template.md

```diff
@@ -6,8 +6,8 @@ Describe Change Here!
 
 # Implementation
 
-* WPT tests: [LINK]()
+* WPT tests:
 * Implementations (link to issue or when done, link to commit):
-   * WebKit: [ISSUE]()
-   * Gecko: [ISSUE]()
-   * Blink: [ISSUE]()
+   * WebKit:
+   * Gecko:
+   * Blink:
```

Comment:
I'm not sure implementation bug links are needed. I've been filing bugs on WebKit and Blink ([example](https://bugs.chromium.org/p/chromium/issues/detail?id=1428763)) as I find them with the related WPT test, and Gecko's WebDriver patch is in source but not yet deployed to WPT.